Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,34 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
 
 
-char percent = 0;
-int currentVoltage = 0;
+int batteryVoltage = 0;
 
 void __attribute__((interrupt, no_auto_psv)) _ADC2Interrupt(void)
 {
 
-    currentVoltage = ADC2BUF0;
+    batteryVoltage = ADC2BUF0;
+        
     IFS1bits.AD2IF = 0;		// Clear the ADC Interrupt Flag
 
 
 }
 
-void initBatterySensor(){
+void initMainBatterySensor(){
     //AN12 is the pin to get the battery information
     TRISBbits.TRISB12 = 1;
-    initVoltageADC();
+    initMainBatteryADC();
 
 }
 
-//float timeRemaining(){
-//    float dP = (currentPercent - lastPercent)/vTimeInterval; //Rate of change (Percent/second)
-//    float time = currentPercent/dP;
-//    return time;
-//}
-
-int getBatteryLevel(){
-    percent = currentVoltage;
-    return percent;
+int getMainBatteryLevel(){
+//return raw voltage value
+    return batteryVoltage;
 }
 
-void initVoltageADC(){
+void initMainBatteryADC(){
```

Comment:
we're not using this method anywhere, thus theres no need for it to be available publically. Add a static property to this method, so that it can only be used within the same file, and remove it from the MainBattery.h file.